main.tf

```diff
@@ -1,6 +1,8 @@
 locals {
   enabled = module.this.enabled
 
+  table_name = var.table_name != null && var.table_name != "" ? var.table_name : module.this.id
```

Comment:
why checking for ""?

Reply:
@jamengual I used the same setup from the s3 bucket module

https://github.com/cloudposse/terraform-aws-s3-bucket/blob/126e8ad4c67b41b606bf6de32fbfdc96365ec522/main.tf#L13

Reply:
`s,bucket_name,table_name,g`

Reply:
no problem, I just wanted to make sure.